dali/operators/reader/loader/video_loader.cc

```diff
@@ -561,6 +563,24 @@ void VideoLoader::read_file() {
 
       file.last_frame_ = frame;
       key = (pkt->flags & AV_PKT_FLAG_KEY) != 0;
+
+      if (key) {
+        last_key_frame = frame;
+      }
+
+      // if decoding hasn't produced any frames after providing 10 frames,
```

Comment:
10 is just a best guess value, although I guess there could be a video when the distance between key frames is smaller, but still the first part of the condition should kick off.

Reply:
Maybe the `10` could be extracted as some kind of constant? `max_num_frames_in_decoder` or something like that?

Reply:
Done